modules/cluster-logging-release-notes-5.0.0.adoc

```diff
@@ -173,7 +173,7 @@ The current release fixes this issue. Now, when a rollover occurs in the `indexm
 
 * Previously, the code that uses environment variables to replace values in the Kibana configuration file did not consider commented lines. This prevented users from overriding the default value of server.maxPayloadBytes. The current release fixes this issue by uncommenting the default value of server.maxPayloadByteswithin. Now, users can override the value by using environment variables, as documented. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918876[*BZ#1918876*])
 
-* Previously, logs were not sent to managed storage when legacy log forwarding was enabled. This happened because the internal generation of the `logforwarding` configuration improperly made a decision for  either `logforwarding` or legacy `logforwarding`. The current release fixes this issue:  Logs are sent to managed storage when the logstore is defined in the `clusterlogging` instance.  Additionally, logs are sent to legacy `logforwarding` when enabled regardless of whether a managed logstore is enabled or not. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921263[*1921263*])
+* Previously, logs were not sent to managed storage when legacy log forwarding was enabled. This happened because the internal generation of the `logforwarding` configuration improperly made a decision for  either `logforwarding` or legacy `logforwarding`. The current release fixes this issue. Now, if the `clusterlogging` instance defines the `logStore`, log forwarding sends the logs to managed storage.  Additionally, logs are sent to legacy `logforwarding` when enabled regardless of whether a managed `logStore` is enabled or not. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1921263[*1921263*])
```

Comment:
If I understand correctly, you went back and updated this after I had reviewed? I hadn't noticed that there was the same/similar section already in the docs. The only thing that I'd say now is that these look like the same bug fix description, but now there are differences. Like "`ClusterLogging` instance" vs. "`clusterlogging` instance."

I'd suggest to decide which is the right way for this bug description and the be consistent between these two sections so that they match.